<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 61, 61);</string>
  </property>
  <widget class="QPushButton" name="aboutus_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(3, 3, 3); background-color: rgb(202, 202, 202);</string>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cliente_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(3, 3, 3); background-color: rgb(202, 202, 202);</string>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(3, 3, 3); background-color: rgb(202, 202, 202);</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titolo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(236, 236, 236); font-size:28pt;</string>
   </property>
   <property name="text">
    <string>EasyParking</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frase">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(236, 236, 236)</string>
   </property>
   <property name="text">
    <string>Sei l'Amministratore o un Dipendente? Fai il Login!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(3, 3, 3); background-color: rgb(202, 202, 202);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
